wiki/People/Nomination_Assessment_Team/en.md

```diff
@@ -0,0 +1,56 @@
+# Nomination Assessment Team
+
+The **Nomination Assessment Team** (***NAT***) are the moderators of the [Beatmap Nominators](/wiki/People/Beatmap_Nominators). They are responsible for reviewing BN applications, creating tests for the applicants and evaluating current members. All NAT members may act as BNs in the modding and ranking process, but they do not have strict activity requirements like BNs.
```

Comment:
@pishifat is there anything I should add/remove from this and next paragraph

Reply:
overseeing vetoes is another NAT responsibility, otherwise good

also is it proper wiki styling/formatting to pluralize an initialism? (BNs)

Reply:
ok and yes

thanks!